src/huggingface_hub/hf_api.py

```diff
@@ -389,6 +390,21 @@ class AccessRequest:
     fields: Optional[Dict[str, Any]] = None
 
 
+@dataclass
+class WatchedItem:
+    """Data structure containing informatin about webhooks watched for updates.
```

Comment:
```suggestion
class WebhookWatchedItem:
    """Data structure containing information about the items watched by a webhook.
```

I'd rename it `WebhookWatchedItem` to avoid confusion with other dataclasses unrelated to webhooks. This change has to be reflected in every occurrence of `WatchedItem`